### DISCUSSION TITLE
Playback streams

### DISCUSSION THREAD

**Discussion by alexandrecastro-bib** [bluenviron/mediamtx#3156] [Questions]
### Question

Hello all  ,

I have setup recording and playback on the server.
Everything is working fine on local machine with queries like this one : 
http://localhost:9996/get?path=cam2hd&start=2024-03-20T14%3A19%3A17.083366%2B01%3A00&duration=60

I only have an issue outside of server when requesting  because clients are requesting http://{mydomain}:9996/get?path=cam2hd&start=2024-03-20T14%3A19%3A17.083366%2B01%3A00&duration=60 HTTPS and i have error : net::ERR_SSL_PROTOCOL_ERROR 

So my question is :  is there a way to have HTTPS on playback API and secure connection  ? 

Thanks in advance 

Regards

**Comment by alexandrecastro-bib** [comment]
Hello dears,

I have no answer on my question. 
Nobody knows why we can't secure requests on playback with the mediamtx config file like we do for WEBRTC ? 

Thanks in advance

Regards

**Comment by rfscheidt** [comment]
hello,

i have no problem with playback:

https://IP:9996/list?path=XXX
https://IP:9996/get?path=XXX&start=2024-03-20T19:58:48.516409Z&duration=105

**Reply by alexandrecastro-bib** [reply]
Thanks for answer. 
did you setup somerhing in config file ? 
could you please show me your config file please ? 
I’m asking it cause author told me it was not implemented yet.
[last question](https://github.com/bluenviron/mediamtx/discussions/3146)

Regards

**Reply by alexandrecastro-bib** [reply]
Hello @rfscheidt ,

I'm still struggling and can't figure out how you managed to set up HTTPS on your playback API like this : https://ip:9996/list?path=XXX 

There is not such configuration in the yaml file.

Could you explain how you did this ? 

Regards

**Comment by rfscheidt** [comment]
when you post a new path, you need to specify:

   "record": false,
   "playback": false

post http://IP:9997/v3/config/paths/add/XXX

{
  "name": "XXX",
  "source" : "rtsp:/IP:10001/cam/realmonitor?channel=1&subtype=0",
   "record": true,
   "playback": true
}

**Reply by alexandrecastro-bib** [reply]
Ok thanks i’ll try this but i have automatic record and playback and all sources setup in config file.

**Comment by aler9** [comment]
moved here: https://github.com/bluenviron/mediamtx/issues/3235
